pkg/Microsoft.NETCore.Platforms/frameworks.json

```diff
@@ -0,0 +1,84 @@
+{
+  "frameworkDefinitions": {
+    ".NETPlatform": {
+      "folderName": "dotnet",
+      "displayName": ".NET Platform",
+      "versions": {
+        "v5.1": {
+          "name": ".NETPlatform,Version=v5.1",
+          "folderName": "dotnet5.1",
+          "displayName": ".NET Platform 5.1"
+        },
+        "v5.2": {
+          "name": ".NETPlatform,Version=v5.2",
+          "folderName": "dotnet5.2",
+          "displayName": ".NET Platform 5.2"
+        },
+        "v5.3": {
+          "name": ".NETPlatform,Version=v5.3",
+          "folderName": "dotnet5.3",
+          "displayName": ".NET Platform 5.3"
+        },
+        "v5.10": {
+          "name": ".NETPlatform,Version=v5.10",
+          "folderName": "dotnet5.10",
+          "displayName": ".NET Platform 5.10"
+        }
+    },
+
+    ".NETFramework": {
+      "folderName": "net",
+      "versions": {
+        "v4.0": {
+          "name": ".NETFramework,Version=v4.0",
+          "folderName": "net40",
+          "displayName": ".NET Framework 4",
+          "profiles": {
+            "Client": {
+              "name": ".NETFramework,Version=v4.0,Profile=Client",
+              "folderName": "net40-client",
+              "displayName": ".NET Framework 4 Client Profile"
+            }
+          }
+        },
+        "v4.5": {
+          "name": ".NETFramework,Version=v4.5",
+          "folderName": "net45",
+          "displayName": ".NET Framework 4.5",
+          "imports": [ ".NETPlatform,Version=v5.1" ]
```

Comment:
Are there rules about order in which these are evaluated if there is more than one?  For example, today you only hit "dotnet*" after looking at all "net*".